docs/source/share_kedro_viz.md

```diff
@@ -23,20 +23,17 @@ When prompted for a project name, you can enter anything, but we will assume `Sp
 
 When your project is ready, navigate to the root directory of the project.
 
-
 ## Publish and share Kedro-Viz automatically
 
+There are two ways to publish and share your Kedro-Viz:
 
-When your project is ready, navigate to the root directory of the project.
-
-There are two ways to publish and share your Kedro-Viz: 
-    1. You can automate the process of publishing and sharing your Kedro-Viz. This section describes the steps for AWS, which is the only cloud provider supported for automation at present. Integration with other cloud providers, namely Azure and GCP, will be added soon. 
-    2. In the absence of automated publish and share for other platforms, there is a [manual, platform-agnostic publish and share process](#platform-agnostic-sharing-with-kedro-viz) described below. You can use the manual process for sharing on static website hosts like GitHub pages, and cloud providers like Azure and GCP.
-
+1. You can automate the process of publishing and sharing your Kedro-Viz. This section describes the steps for AWS, which is the only cloud provider supported for automation at present. Integration with other cloud providers, namely Azure and GCP, will be added soon.
```

Comment:
Do we need to update this since we are adding Azure and GCP?

Reply:
I added a note for 7.2.0 separately as I was not sure how to describe for users using viz sharing before 7.2.0